### PR TITLE
Gra 905 freebsd - use wgctl 

### PR DESCRIPTION
**Pull request by mattkasun**
note:  must be built on freebsd machine.  cross-compiling will not work
[issue on wireguard repo](https://github.com/WireGuard/wgctrl-go/issues/130)

### CODE REVIEW

File: wireguard/wireguard_freebsd.go [hunk 1]
```diff
@@ -124,47 +68,114 @@ func create(nc *NCIface) error {
...
 }
 
-func strip() error {
-	in, err := os.Open(config.GetNetclientPath() + "netmaker.conf")
-	if err != nil {
-		return err
-	}
-	defer in.Close()
-	out, err := os.Create(os.TempDir() + "/netmaker.conf")
+// netlink.SetMTU - set MTU for netmaker interface
```
Comment (0xdcarns) [inline wireguard/wireguard_freebsd.go:74]: incorrect comment - NCIface.SetMTU

File: wireguard/wireguard_freebsd.go [hunk 2]
```diff
@@ -124,47 +68,114 @@ func create(nc *NCIface) error {
...
+		return fmt.Errorf("error setting mtu %w", err)
 	}
-	writer.Flush()
 	return nil
 }
 
-func apply(n *config.Node, c *wgtypes.Config) error {
-	return nil
-}
+// Configure does nothing on freebsd, all configuration is done by NCIface.Create()
```
Comment (0xdcarns) [inline wireguard/wireguard_freebsd.go:88]: is there a purpose to keeping the commented out functions?

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Review by theguy951357 — APPROVED** [review]

File: wireguard/wireguard_freebsd.go (hunk 2 above)
Comment (mattkasun) [reply wireguard/wireguard_freebsd.go:88]: removed

**Comment by 0xdcarns** [comment]
merge conflicts need resolution

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**